modules/persistent-storage-csi-sc-managing-cli.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-managing-cli_{context}"]
+= Managing the default storage class using the CLI
+
+.Prerequisites
+* Administrator rights.
+
+.Procedure
+
+To manage the storage class using the CLI, run the following command:
```

Comment:
According to the mod docs [procedure template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc): If you have only one prerequisite, list it as a single bullet point